bin/test-boot

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+MEM=${MEM:-512}
+GUESTARCH="${GUESTARCH:-$(echo $(basename $IMG)| cut -d'-' -f3)}"
+VER=$(echo $(basename $IMG)| cut -d'-' -f2)
+
+echo '{"instance-id": "9068aef2-213e-4e43-830f-accdbadde897"}' > meta-data
+{ echo '#!/bin/sh'; echo 'echo Hello from inside'; echo 'poweroff -f'; } > user-data
+
+
+cloud-localds -d qcow2 seed.img user-data meta-data
+
+qemu-img create -f qcow2 -b "$IMG" -F qcow2 disk1.img >/dev/null 2>&1
+
+EXTRA_OPTS=
+
+case $GUESTARCH in
+    arm)
+    	MACHINE=virt
+    	EXTRA_OPTS="-kernel $RELEASE_DIR/cirros-$VER-arm-kernel -initrd $RELEASE_DIR/cirros-$VER-arm-initramfs" ;;
+    aarch64)
+    	MACHINE=virt
+    	EXTRA_OPTS="-cpu cortex-a57 -kernel $RELEASE_DIR/cirros-$VER-aarch64-kernel -initrd $RELEASE_DIR/cirros-$VER-aarch64-initramfs" ;;
+    ppc64)
+    	MACHINE=pseries ;;
+    ppc64le)
+    	MACHINE=pseries
+    	GUESTARCH=ppc64 ;;
+    powerpc)
+    	# No idea how to run it
+    	exit 1 ;;
+    i386|x86_64)
+    	MACHINE=pc ;;
+esac
+
```

Comment:
I'd like to see in the logs the command that is executed.
there are 2 ways to easily do that:

 a.) set -x (which I generally hate, but for the last line of a script  like this its not that bad)
 b.) set and echo

        set -- qemu-system-$GUESTARCH -m $MEM -machine $MACHINE \ ....
        echo "executing guest with:" "$@"
        "$@"